flow/envs/base_env.py

```diff
@@ -879,16 +881,20 @@ def terminate(self):
         Should be done at end of every experiment. Must be in Env because the
         environment opens the TraCI connection.
         """
-        print(
-            "Closing connection to TraCI and stopping simulation.\n"
-            "Note, this may print an error message when it closes."
-        )
-        self.traci_connection.close()
-        self.scenario.close()
-
-        # close pyglet renderer
-        if self.sim_params.render in ['gray', 'dgray', 'rgb', 'drgb']:
-            self.renderer.close()
+        try:
+            print(
+                "Closing connection to TraCI and stopping simulation.\n"
+                "Note, this may print an error message when it closes."
+            )
+            self.traci_connection.close()
+            self.scenario.close()
+
+            # close pyglet renderer
+            if self.sim_params.render in ['gray', 'dgray', 'rgb', 'drgb']:
+                self.renderer.close()
+        except FileNotFoundError:
```

Comment:
when does this happen?

Reply:
If an user manually closes the env, `scenario.close()` will fail since all the net files and cfg files are already gone. Thus the `FileNotFoundError`.

Reply:
kk, sounds good